make-release.sh

```diff
@@ -148,15 +148,14 @@ else
   # bump the z digit
   [[ $VERSION =~ ^([0-9]+)\.([0-9]+)\.([0-9]+) ]] && BASE="${BASH_REMATCH[1]}.${BASH_REMATCH[2]}"; NEXT="${BASH_REMATCH[3]}"; # for VERSION=7.25.2, get BASE=7.25, NEXT=2 [DO NOT BUMP]
   NEXTVERSION_Z="${BASE}.${NEXT}"
-  bump_version ${NEXTVERSION_Z} ${BASEBRANCH}
+  bump_version ${NEXTVERSION_Z} "master"
```

Comment:
Do we want to hardcode it down here or set a BASEBRANCH variable as on line 113? https://github.com/eclipse/che-docs/pull/1842/files#diff-be36842270d47fd0f7dd297abbdf5c5ef7916efe78fd4617f5ca6a5fdcba421cR113

Reply:
basebranch meant something different, as it cane be major or bugfix branch. So I'll rework it, since we have to push in both branches for both release cases